components/FormInput/FormInput.js

```diff
@@ -10,57 +11,65 @@ export default class FormInput extends Component {
     this.handleBlur = this.handleBlur.bind(this);
 
     this.state = {
-      active: !!props.value,
       focus: false,
+      id: uniqueId('forminput'),
```

Comment:
Lol, yes. Silly me. I have seen two different patterns for this however. One as state, [another as a property on the class](https://github.com/PactCoffee/loggins/blob/feature/my-ui/components/RadioGroup/Radio.js). Let's pick one and stick to it.